### DISCUSSION TITLE
`Gitter <-> Matrix <-> Discord` integration (Bring back Gitter)

### DISCUSSION THREAD

**Discussion by sitiom** [chocolatey/home#149] [Ideas]
## Is Your Feature Request Related To A Problem? Please describe.
I have just suggested this in Scoop (https://github.com/ScoopInstaller/Scoop/issues/4831). I think the same thing applies here, so I'll reiterate some points here.

[Matrix](http://matrix.org/) is a great way to unify various other chat platforms via [bridges](https://matrix.org/bridges/). And for some time now, Gitter has become [Matrix native](https://element.io/blog/gitter-goes-native/). I have been accessing the Chocolatey Gitter rooms this way until the announcement of moving to Discord. I also believe other Gitter and Matrix users are unhappy about this, moving to a closed and proprietary platform where information is not easily accessible publicly (See [Discord is a black hole for information.](https://knockout.chat/thread/33251/1)). The best way to meet both ends is to connect Chocolatey's Discord server to Matrix Rooms to leverage the benefit of Gitter's indexable public message history.

## Describe The Solution. Why is it needed?
See https://github.com/elves/elvish. They were able to set it up successfully with a `Gitter <-> Matrix <-> Telegram` integration. Discord isn't anything different.

Here's what you can do:
1. Create a Matrix Space for Chocolatey and the corresponding rooms that map to Gitter and Discord rooms/channels.
2. Ask Gitter support to plumb the Gitter rooms to the corresponding Matrix Rooms.
3. Bridge Matrix Rooms to the corresponding Discord channels via the Discord bridge.
4. Voila! Everyone is happy, and all messages are publicly accessible and indexed by search engines.

## Additional Context.

N/A

## Related Issues

- Issue #140

**Comment by sitiom** [comment]
Why is this moved to a discussion?

**Reply by pauby** [reply]
A discussion is a more appropriate place for this.

**Comment by rollingmoai** [comment]
+1 for this, I don't like using Discord as much as possible.

**Comment by pauby** [comment]
Just to manage expectations on this, there are no plans to implement Matrix bridges.

In addition, Chocolatey Gitter channels will be closed soon.
